File: Py/8494_cm.py
import sys

# n, m 입력 및 노드 입력
n, m = map(int, sys.stdin.readline().strip().split(' '))
node_location = [list(map(int, sys.stdin.readline().strip().split(' '))) for i in range(m)]

# 2차원 배열 초기화
arr = [[0] * n for _ in range(n)]

# (노드의 위치값-1)을 위치로 배열에 1로 맵핑, x & y 좌표를 반대로 해서 반대로 맵핑
for i in range(m):
	x_loc = node_location[i][0] - 1
	y_loc = node_location[i][1] - 1

	arr[x_loc][y_loc] = 1
	arr[y_loc][x_loc] = 1

for i in arr:
		for j in i:
			print(j, end = ' ')
		print()

File: JS/8494_cm.js
function noDirectedGraph(n, m, nodeLocation) {

	// let arr = [];

	// for (let i = 0; i < n; i++) {
	// 	arr.push(Array(n).fill(0));
	// }

	let arr = new Array(n).fill(0).map(() => new Array(n).fill(0));

	for (let i = 0; i < m; i++) {
		let xLoc = nodeLocation[i][0] - 1;
		let yLoc = nodeLocation[i][1] - 1;

		arr[xLoc][yLoc] = 1;
		arr[yLoc][xLoc] = 1;
	}

	for (let i = 0; i < n; i++) {
		let r = '';

		for (let j = 0; j < n; j++) {
			r += arr[i][j] + ' ';
		}
		console.log(r.trim());
	}
}

n = 3;
m = 3;
l = [[1, 2], [1, 3], [2, 3]];

noDirectedGraph(n, m, l);